docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -274,6 +276,12 @@ fields:
     datatype: file
     accept: |
       ".pdf, application/pdf, .docx, application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+  - Re-process and auto-identify form fields (PDFs, removes existing fields): yes_recognize_form_fields
```

Comment:
I played around with 
```yaml
js show if: |
  val("template_upload") !== null && val("template_upload").endsWith(".pdf")
```
to avoid a bit of the extra "PDF only"  in the explainer, but couldn't get it to work (worked fine in the console, but the show if part wasn't).  But it's not a big deal, since it doesn't break things if you check it with an uploaded docx.